pulp_python/app/models.py

```diff
@@ -72,6 +72,18 @@ class Meta:
             'filename',
         )
 
+    def __str__(self):
+        """
+        Same as the base Content one, except that it adds the python package type at the end.
+        e.g. <PythonPackageContent: shelf-reader (whl)>
+        """
+        try:
+            # if we have a name, use it
+            return '<{}: {} ({})>'.format(self._meta.object_name, self.name, self.packagetype)
+        except AttributeError:
+            # if we don't, use the pk
+            return '<{}: pk={}>'.format(self._meta.object_name, self.pk)
+
```

Comment:
This isn't necessary, it just makes debugging nicer.  We can delete it if you want.

Reply:
Not quite what I meant.  If you are hacking around using a REPL or an interactive debugger, str(model) is what you're getting when you look at the value of a variable containing the model or queryset containing the models. 

But I'll remove it anyway because it's usefulness doesn't scale up all that well.

Reply:
I meant the entire block, but you're right that we can remove the try/except